src/main/java/edu/stanford/protege/webprotegeeventshistory/EntityUpdateFailedEvent.java

```diff
@@ -0,0 +1,36 @@
+package edu.stanford.protege.webprotegeeventshistory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.stanford.protege.webprotege.common.ChangeRequestId;
+import edu.stanford.protege.webprotege.common.EventId;
+import edu.stanford.protege.webprotege.common.ProjectEvent;
+import edu.stanford.protege.webprotege.common.ProjectId;
+
+import javax.annotation.Nonnull;
+
+public record EntityUpdateFailedEvent(@JsonProperty("projectId") ProjectId projectId,
+                                      @JsonProperty("eventId") EventId eventId,
+
+                                      @JsonProperty("entityIri") String entityIri,
+                                      @JsonProperty("changeRequestId")ChangeRequestId changeRequestId) implements ProjectEvent {
+
+    public static String CHANNEL = "webprotege.api.EntityUpdateFailed";
```

Comment:
[question]: this event from rabbit can be handled multiple times by multiple services? basically everyone who subscribed to the channel will receive the notification right?